hack/make/.ensure-frozen-images

```diff
@@ -4,7 +4,7 @@ set -e
 # this list should match roughly what's in the Dockerfile (minus the explicit image IDs, of course)
 images=(
 	busybox:latest
-	hello-world:latest
+	hello-world:frozen
```

Comment:
All else fails meaning that pull directly by image ID failed? If the image is intended to be pinned at a particular version, I don't see why we would want to `docker pull <image>` or `docker pull <image>:<tag>`.  Given the current circumstances though I will explore another way to fix this part of the issue.